lib/middleware/authorize.js

```diff
@@ -0,0 +1,24 @@
+const Item = require('../models/Item');
+
+module.exports = async (req, res, next) => {
+  try { 
+    if(req.method === 'PUT') {
```

Comment:
you don't need to wrap these in `if` conditions -- since the `authenticate` middleware is only running on the PUT and DELETE methods and your code inside is the same, you can just simplify this to 
```js
 const item = await Item.getById(req.params.id);
      if (req.user.id !== item.user_id) {
        throw new Error('Not your item!');
      }
```